### ISSUE TITLE
Rotated printed data after printing when the print preview is ok. 

### ISSUE THREAD

**Issue by themachho** [sumatrapdfreader/sumatrapdf#2353]
Hello. Im having trouble with printing a simple pdf with sumatra.exe on windows 64bit.

I started using a tool that calls SumatraPDF.exe and notice this issue. Later i downloaded the portable version of Sumatra to execute it standalone and had the same issue.

This is the print preview on SumatraPDF v3.3.3 64-bit. The file it's a label of 6''x5''. Its landscape and the color boxes are on the edges of the label.  [Label 6x5.pdf](https://github.com/sumatrapdfreader/sumatrapdf/files/8089631/Label.6x5.pdf)

![print_preview](https://user-images.githubusercontent.com/92321864/154503079-700b7749-d7d2-48a9-b886-560c91280094.png)

This is the result when i print with Adobe Reader. The result/print is the expected:

![Adobe_print](https://user-images.githubusercontent.com/92321864/154503334-aebad56b-7e86-48d8-af6d-9e378e8f0ae8.jpg)

As you can see, the color boxes are on the edges and the preview matches the print. 
But when i print with SumatraPDF this is what i get:

![SumatraPDF_print](https://user-images.githubusercontent.com/92321864/154503498-f5239d33-79dc-48c5-90aa-2d8ab9fd2625.jpg)

The printer settings are the same, but the print is rotated 90° for some reason, even when landscape is selected and the print preview is what should be printed.

Any ideas why?

**Comment by themachho** [comment]
The tool is [pdf-to-printer](https://www.npmjs.com/package/pdf-to-printer) and cant access the executed line, but it happens with own cmd lines `.\SumatraPDF.exe -print-to "MYPRINTER" -print-settings "fit,portrait"  ..\label6x5.pdf`. 

I also tried to change the default printer orientation with no success. Tested fit, shrink, noscale, portrait/landscape and same issue. I will continue to investigate a solution, but can't find anything immediate.

**Comment by GitHubRulesOK** [comment]
PDF-to-printer is an old clone (thus unsupported) and may not have same commands as current pre-release which has paperkind thus much easier for lable printing see  https://github.com/sumatrapdfreader/sumatrapdf/issues/1812

**Comment by GitHubRulesOK** [comment]
![l9wNS 1](https://user-images.githubusercontent.com/12754913/156901876-68b59b5b-655d-415b-b642-cde98eb1ba66.png)
 If your FORMS are set correctly it should be no problem printing full media at any orientation so here I defined 6x5 and 5x6 and that is the measured output.

It may help to try a different small format specialist driver, thus for labels ["Bartender Seagull"](https://www.seagullscientific.com/) drivers (if available for that printer model) are often recommended.

 HOWEVER it is not a simple task for small formats and usually requires much overriding of windows default settings. Most important DO NOT use -silent switch, especially when print serving.
![ZKBR1 1](https://user-images.githubusercontent.com/12754913/156901919-98fdd96c-d2c1-4894-934c-2d489e43b424.png)

Note I had to add a 6x5 Form in Windows print management, and obscurely that is actually defined as 5 inches wide by 6 inches high even though it says 6.00x5.00 ! since you want it to be applied rotated as Landscape. I then have to ensure the driver is set to Landscape by default, it is likely to fail if needing headless rotation. So now I have my Media structure as paperkind=138 (the number will be different on every machine and I could use setting `paper=6x5,` which did work too) numbering is more reliable for some drivers media lists. Also since we know your image is oversized I need to add shrink. By far it is easier to use Acrobats defaults rather than Microsofts, and Acrobat can be left running in standby (/T) so it does not need to slowly reload for each print.
